plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1169,24 +1170,23 @@ protected TestView createSleepingView(Duration minimalSleepDuration)
     @Test
     public void testDeleteWithBigintEqualityPredicate()
     {
-        skipTestUnlessSupportsDeletes();
-        String tableName = "test_delete_" + randomTableSuffix();
-        assertUpdate("CREATE TABLE " + tableName + " AS SELECT * FROM region", 5);
-        assertUpdate("DELETE FROM " + tableName + " WHERE regionkey = 1", 1);
-        assertQuery(
-                "SELECT regionkey, name FROM " + tableName,
-                "VALUES "
-                        + "(0, 'AFRICA'),"
-                        + "(2, 'ASIA'),"
-                        + "(3, 'EUROPE'),"
-                        + "(4, 'MIDDLE EAST')");
-        assertUpdate("DROP TABLE " + tableName);
+        skipTestUnless(hasBehavior(SUPPORTS_CREATE_TABLE_WITH_DATA) && hasBehavior(SUPPORTS_ROW_LEVEL_DELETE));
```

Comment:
is `skipTestUnlessSupportsDeletes` still used?

Reply:
Sadly for Kudu since it doesn't use BaseConnectorTest and uses ATDQ instead which doesn't have these flags. 😞